content/pages/education/gi-bill.md

```diff
@@ -39,7 +39,7 @@ majorlinks:
     - url: /education/gi-bill/survivors-dependent-assistance/
```

Comment:
these changes were pulled from [this](https://github.com/department-of-veterans-affairs/vets-website/pull/7947) content update